### PR TITLE
Create Tweet_call_for_school.md

### PR DESCRIPTION
**Pull request by giulia-pacchioni**
Proposed tweets for our 'call for summer schools'

### CODE REVIEW

**Comment by greplova** [comment]
Thank you for suggesting the tweets Giulia! I like! Can I have thumbs up from someone and maybe just go ahead and tweet? The next long range is still a week away and summer started already, so I think there's no reason to wait.

File: texts/Tweet_call_for_school.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+Tweet 1:
+Grad students and postdocs! Have you thought of organizing a #SummerSchool? It's a great experience for young researchers & very valuable for attendees. We would be delighted to help and support you in the organization. Please RT to spread the word!
+Tweet 2:
+The Princeton Summer School on condensed matter (https://pccm.princeton.edu/education/psscmp2020, @pccm_mrsec)  for example was organized by students and was a huge success. It would be great to see more like that this summer!
+
+(I added the tweet2 so people can have an example of what type of school we have in mind)
```
Comment (akhmerov) [inline texts/Tweet_call_for_school.md:6]:
```suggestion
```
(comments shouldn't go into the repository)

**Comment by akhmerov** [comment]
Looks good.

**Comment by giulia-pacchioni** [comment]
Yep, could be a good time to tweet it so we leave a  few days before the next LRC tweet.

**Pull request merged**